### ISSUE TITLE
The Felon shield gun looks like a disruptor lightning weapon 

### ISSUE THREAD

**Issue by Sgathec** [ZeroK-RTS/Zero-K#3663]

**Comment by sprunk** [comment]
Not really, but feel free to propose an alternative. The zap could probably just be red or blue instead of purple.

**Comment by Anarchid** [comment]
The zap should be the same color as the shield currently is. This is achievable by replacing the engine Lightning rendering with a custom shader on top of custom rendering for the weapon. 

For what it's worth, i think Lightning looks horrible and should be replaced with the above whenever anyone bothers to write such a particle and a shader, similar to how was done to shields.

**Comment by GoogleFrog** [comment]
The purple colour has become a problem and the effect looks fairly bad. Dynamic lights help slightly. I like the descriptiveness of the effect, in that it accurately reflects the fire-blocking behaviour of Felon and the visual gives a good indication of how much variable damage it is dealing. The easiest way to retain these two aspects is to make a new beam image.
